app/worker/parser.py

```diff
@@ -14,6 +14,10 @@ def parser(csvfile):
 
     for row in read_file:
         row = {k: unicode(v, "utf-8", errors='ignore') for k, v in row.items()}
+        items = row.items().value()
```

Comment:
This should be simplified down to `row = {k: v.strip() for k, v in row.items()}`